packages/api-client/src/api/cart/addItems/index.ts

```diff
@@ -0,0 +1,24 @@
+import queryString from 'query-string';
+import BigCommerceEndpoints from '../../../helpers/endpointPaths';
+import {
+  BigcommerceIntegrationContext,
+  AddLineItemsParameters,
+  AddLineItemsResponse
+} from '../../../types';
+
+export async function addItems(
+  context: BigcommerceIntegrationContext,
+  params: AddLineItemsParameters
+): Promise<AddLineItemsResponse> {
+  const { cartId, data, include } = params;
+
+  return await context.client.post(
```

Comment:
Just a small thing, `await` in `return await` is redundant because you're returning the promise anyway
```suggestion
  return context.client.post(
```

Reply:
they are not the same:
https://stackoverflow.com/questions/38708550/difference-between-return-await-promise-and-return-promise
there is also an eslint rule to require await for every async function to make refactoring more error prune: [require-await](https://eslint.org/docs/rules/require-await)

Reply:
Hm didn't realise the case with try/catch blocks, in my previous project there was a rule to remove every await from return await ;p But good to know that! Then I'm resolving the conversation